packages/ui/src/lib/components/Stage/components/AnnotationLayer/types.ts

```diff
@@ -18,7 +18,7 @@ export interface AnnotationsLayerProps {
 /**
  * The properties for an individual annotation drawing layer
  */
-export interface AnnotationLayer {
+export interface AnnotationLayerData {
```

Comment:
Sorry I didn't mention this before in the previous. Can we add opacity to this? I'm thinking it'll give it some flexibility to draw "zones"